chia/pools/pool_wallet.py

```diff
@@ -448,14 +449,24 @@ async def create_new_pool_wallet_transaction(
         )
         return standard_wallet_record, p2_singleton_puzzle_hash, launcher_coin_id
 
-    async def sign(self, coin_spend: CoinSpend) -> SpendBundle:
+    async def sign(self, coin_spend: CoinSpend, fee: uint64) -> SpendBundle:
```

Comment:
This makes no sense to me that a function called `sign` also generates an unsigned transaction. The fee should be included in the initial unsigned transaction.

Reply:
That is an excellent point. The code that creates this CoinSpend is in the driver code `chia/pools/pool_puzzles.py:create_travel_spend` - could you or Quex add a fee parameter here, please?